nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/CsrfCookieRequestMatcher.java

```diff
@@ -35,6 +33,6 @@
      */
     @Override
     public boolean matches(final HttpServletRequest httpServletRequest) {
-        return WebUtils.getCookie(httpServletRequest, DEFAULT_CSRF_COOKIE_NAME) != null;
+        return WebUtils.getCookie(httpServletRequest, SecurityCookieName.AUTHORIZATION_BEARER.getName()) != null;
```

Comment:
Just for my own understanding - is the jwt token value also used as the CSRF prevention token value?

Reply:
That's correct. Changes in NIFI-7870 introduced sending the JWT in an HTTP session cookie, but retained sending the JWT in the Authorization header. This helps mitigate potential CSRF issues since the browser automatically sends Cookies, but does not send the Authorization header unless the NiFi UI includes it as part of the request.

Reply:
Thanks for confirming!